includes/admin/payments/orders.php

```diff
@@ -602,59 +668,82 @@ function edd_order_details_adjustments( $order ) {
-							'show_option_none' => false,
-							'chosen'           => true,
-						) ); // WPCS: XSS ok. ?>
+						<label for="edd_order_add_adjustment_select" class="edd-form-group__label edd-order-details-label-mobile"><?php echo esc_html_x( 'Adjustment To Add', 'order details select adjustment to add - mobile', 'easy-digital-downloads' ); ?></label>
+					<div class="edd-form-group__control">
+						<?php
+						echo EDD()->html->select(
+							array(
+								'name'             => 'edd-order-add-adjustment-select',
+								'id'               => 'edd-order-add-adjustment-select',
+								'class'            => 'edd-order-add-adjustment-select edd-form-group__input',
+								'options'          => array(
+									''         => '', // Empty  option needed to display placeholder.
+									'discount' => __( 'Discount', 'easy-digital-downloads' ),
+									'credit'   => __( 'Credit', 'easy-digital-downloads' ),
+								),
+								'placeholder'      => __( 'Choose an Adjustment Type', 'easy-digital-downloads' ),
+								'show_option_all'  => false,
+								'show_option_none' => false,
+								'chosen'           => true,
+							)
+						); // WPCS: XSS ok.
+						?>
+					</div>
 					</li>
 				</ul>
-
+			</div>
+			<div class="edd-form-group">
 				<ul>
 					<li class="discount" style="display: none;">
+						<label for="edd_order_add_discount_select" class="edd-form-group__label screen-reader-text"><?php esc_html_e( 'Discounts', 'easy-ditigal-downloads' ); ?></label>
+					<div class="edd-form-group__control">
 						<?php
-						$d = edd_get_discounts( array(
-							'fields' => array( 'code', 'name' ),
-							'number' => 100,
-						) );
+						$d = edd_get_discounts(
+							array(
+								'fields' => array( 'code', 'name' ),
+								'number' => 100,
+							)
+						);
 
 						$discounts = array();
 
 						foreach ( $d as $discount_data ) {
 							$discounts[ $discount_data->code ] = esc_html( $discount_data->name );
 						}
-
-						echo EDD()->html->discount_dropdown( array(
-							'name'             => 'edd-order-add-discount-select',
-							'class'            => 'edd-order-add-discount-select',
-							'id'               => 'edd-order-add-discount-select',
-							'chosen'           => true,
-							'selected'         => false,
-							'show_option_all'  => false,
-							'show_option_none' => false,
-						) );  // WPCS: XSS ok.
+						echo EDD()->html->discount_dropdown(
+							array(
+								'name'             => 'edd-order-add-discount-select',
+								'class'            => 'edd-order-add-discount-select edd-form-group__input',
+								'id'               => 'edd-order-add-discount-select',
+								'chosen'           => true,
+								'selected'         => false,
+								'show_option_all'  => false,
+								'show_option_none' => false,
+							)
+						);  // WPCS: XSS ok.
 						?>
+					</div>
 					</li>
 				</ul>
-
+			</div>
+			<div class="edd-form-group">
 				<ul>
+				<div class="edd-form-group">
```

Comment:
Is this duplication of line 732 intentional?

Also, I know the `ul/li` are inherited from previous code, but am not sure they are necessary. The `credit` class and inline style are likely necessary for the JS, but seems like the two text input fields could be logically grouped another way than a list with one item, maybe a `fieldset` along the lines of the potential address grouping?

Reply:
This one was intentional, though I'm open to suggestions.  I wasn't sure how to group these since it's technically only one `li` with two input fields and it seemed wrong to split up the `ul` into two `form-group`s.  

I'd love to hear @spencerfinnell s thoughts on changing this from a `ul/li` to ... something else.

Reply:
Probably separate `.edd-form-group` inside each `li`, but since this will be replaced with the new Order Item management UI soon, we don't need to worry about it.